docs/content/en/docs/getting-started/optional/api-server-extra-args.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "API Server Extra Args"
+linkTitle: "API Server Extra Args"
+weight: 60
+description: >
+  EKS Anywhere cluster yaml specification API Server Extra Args reference
```

Comment:
Nit: EKS Anywhere cluster yaml specification for Kubernetes API Server Extra Args reference